src/Place/ReadModel/History/HistoryProjector.php

```diff
@@ -0,0 +1,218 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\ReadModel\History;
+
+use Broadway\Domain\DateTime as BroadwayDateTime;
+use Broadway\Domain\DomainMessage;
+use Broadway\Domain\Metadata;
+use Broadway\EventHandling\EventListenerInterface;
+use CultuurNet\UDB3\Cdb\EventItemFactory;
+use CultuurNet\UDB3\Event\ReadModel\DocumentRepositoryInterface;
+use CultuurNet\UDB3\Event\ReadModel\History\Log;
```

Comment:
This `Log` class is part of the `Event` namespace. Can you move it so it's in for example `CultuurNet\UDB3\History`?